tests.yml

```diff
@@ -3,7 +3,6 @@
     - role: standard-test-beakerlib
       tags:
         - classic
-        - container
```

Comment:
Why we can't add tags here 
https://fedoraproject.org/wiki/CI/Tests
  tags:
  - atomic
  - classic
  - container

Why we need to duplicate them ?